spec/verkle/executionwitness.go

```diff
@@ -59,15 +60,26 @@ func (s *SuffixStateDiff) UnmarshalJSON(input []byte) error {
 	if err := json.Unmarshal(input, &ssd); err != nil {
 		return fmt.Errorf("error unmarshalling JSON SuffixStateDiff: %w", err)
 	}
-	var cleanstring = strings.TrimPrefix(ssd.Suffix, "0x")
-	if len(cleanstring)%2 == 1 {
-		cleanstring = "0" + cleanstring
-	}
-	suffixbytes, err := hex.DecodeString(cleanstring)
```

Comment:
this would cause a value above 100, for example 128, to be interpreted as `[1, 40]`